File: Selenium/Activities/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls")
    print(f"The page title is : {driver.title}")
	
    txtBox = driver.find_element(By.XPATH,"//input[@id = 'textInput']")
    enableInputButton  =  driver.find_element(By.XPATH,"//button[text() = 'Enable Input']")
    print(f"Is the text box visible on the page ? : {txtBox.is_displayed()}")
    print(f"If the textBox is enabled : {txtBox.is_enabled()}")
    enableInputButton.click()
    print(f"If the textBox is enabled : {txtBox.is_enabled()}")
    driver.quit()

File: Selenium/Activities/Activity7.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity7 {

	public static void main(String[] args) {

			WebDriver driver = new FirefoxDriver();
			driver.get("https://training-support.net/webelements/dynamic-controls");
			System.out.println("The page title is : "+ driver.getTitle());
			
			WebElement txtBox = driver.findElement(By.xpath("//input[@id = 'textInput']"));
			WebElement enableInputButton  =  driver.findElement(By.xpath("//button[text() = 'Enable Input']"));
			System.out.println("Is the text box visible on the page ? : " + txtBox.isDisplayed());
			
			System.out.println("If the textBox is enabled : " +txtBox.isEnabled());
			
			enableInputButton.click();
				
			System.out.println("If the textBox is enabled : " +txtBox.isEnabled());
			
			driver.quit();

	}

}
